packages/manager/modules/nasha/src/dashboard/partition/partition.routing.js

```diff
@@ -4,24 +4,21 @@ export default /* @ngInject */ ($stateProvider) => {
   const stateName = 'nasha.dashboard.partition';
   const taskTrackerStateName = `${stateName}.task-tracker`;
 
-  const goToEditResolve = ['description', 'name', 'size'].reduce(
-    (resolves, id) => {
-      const capitalizedId = `${id[0].toUpperCase()}${id.slice(1)}`;
-      return {
-        ...resolves,
-        [`goToEdit${capitalizedId}`]: /* @ngInject */ (
-          $state,
+  const goToEditResolve = ['description', 'size'].reduce((resolves, id) => {
```

Comment:
This looks highly complicated for the wanted result

Reply:
:+1: